addon/mixins/authenticated-route-mixin.js

```diff
@@ -1,21 +1,15 @@
 import Ember from 'ember';
-import Cache from 'ember-gdrive/lib/local-cache';
+import { cacheLoginHint } from 'ember-gdrive/lib/login-hint';
 import AuthenticatedRouteMixin from 'simple-auth/mixins/authenticated-route-mixin';
 
 export default Ember.Mixin.create(AuthenticatedRouteMixin, {
-  cacheLoginHint: function() {
-    var userId = this.get('session.id');
-    var docId = this.get('documentSource.document.id');
-    var cache = new Cache('document_login_hint');
-
-    cache.set(docId, userId);
+  model: function (params) {
+    return this.get('documentSource').load(params.document_id);
   },
 
-  model: function (params) {
-      var route = this;
-      return this.get('documentSource').load(params.document_id).then(function (doc) {
-        route.cacheLoginHint();
-        return doc;
-      });
+  afterModel: function (document, transition) {
+    var userId = this.get('session.secure.id');
```

Comment:
I missed a change in `ember-simple-auth`. The payload returned by a successful authentication request is no longer stored directly in the session object. It's now stored as a `session.secure` property, so that's where the user id is now.